crates/sui-graphql-rpc/src/client/mod.rs

```diff
@@ -24,6 +24,20 @@ pub enum ClientError {
         usage_name: String,
         usage_value: Value,
     },
+    #[error("{item_type} at pos {idx} must not be empty")]
+    InvalidEmptyItem { item_type: String, idx: usize },
+    #[error("Variable {var_name} previously defined {var_type_prev} vs {var_type_curr}")]
```

Comment:
This message is slightly confusing. Does this try to say that a previous variable with the same name was defined, which is not allowed?

Reply:
Indeed, fixed now.